compileall2.py

```diff
@@ -200,6 +208,10 @@ def compile_file(fullname, ddir=None, force=False, rx=None, quiet=0,
         if mo:
             return success
 
+    if limit_sl_dest is not None and os.path.islink(fullname):
+        if Path(limit_sl_dest) not in Path(fullname).resolve().parents:
```

Comment:
Should `limit_sl_dest` be resolved as well?
Is it allowed to pass a relative path as limit_sl_dest?

Reply:
It should, you are right. A relative path does not make sense here.

Reply:
Fixed.